frontends/p4/coreLibrary.h

```diff
@@ -30,6 +30,8 @@ enum class StandardExceptions {
     OverwritingHeader,
     HeaderTooShort,
     ParserTimeout,
+    AssertError,
```

Comment:
This is a big assumption: that we are changing code.p4. 
I would rather expect that assert and assume are library functions - at least for a while.
Putting them in core.p4 may break programs that happen to use identifiers with these names.
This is a subject for the LDWG.